packages/core/tests/storage.test.ts

```diff
@@ -91,38 +63,53 @@ describe("core/storage", () => {
     } catch (err: any) {
       expect(err.message).to.include("upload failed");
     }
+    try {
+      await storage.store(file as any);
+      throw new Error("wrong error");
+    } catch (err: any) {
+      expect(err.message).to.include("not found");
+    }
     const request = await storage.store(file as any);
     expect(request.id).to.equal("fakeId");
     expect(request.cid).to.deep.equal({ "/": "fakeCid" });
-    expect(request.status_code).to.equal("Batching");
+    expect(request.status).to.equal("BATCHING");
   });
 
   test("should be able to store some data using streams", async () => {
     const formData = new FormData();
     formData.set("file", "Hello, world!");
-    const encoder = new Encoder(formData);
+    const encoder = new FormDataEncoder(formData);
     const request = await storage.store(Readable.from(encoder) as any, {
       headers: encoder.headers,
     });
 
     expect(request.id).to.equal("fakeId");
     expect(request.cid).to.deep.equal({ "/": "fakeCid" });
-    expect(request.status_code).to.equal("Batching");
+    expect(request.status).to.equal("BATCHING");
   });
+});
 
+describe("core/storage", () => {
+  beforeEach(async () => {
+    const { signer, kid } = createSigner();
+    const aud = "provider";
+    const { token } = await createToken(signer, { kid }, { aud });
+    storage = create({ token, host: "https://broker.staging.textile.dev" });
+  });
   test("should be able to get status of some data", async () => {
-    const file = new File(["Hello, world!"], "welcome.txt", {
-      type: "text/plain",
-      lastModified: new Date().getTime(),
-    });
-
-    // Test global fallback
-    (globalThis as any).FormData = FormData;
-
-    const { id } = await storage.store(file as any);
+    const { request, deals } = await storage.status(
+      "01fdfw1zggv7esyxja69r6rf5h"
```

Comment:
just some random deal data, eventually this might be cleaned up, and our tests will break... I'm ok with that for now